src/xo-app/dashboard/health/index.js

```diff
@@ -161,7 +177,19 @@ const ALARM_COLUMNS = [
   },
   {
     name: _('alarmContent'),
-    itemRenderer: message => message.body,
+    itemRenderer: ({ formatted }) => isString(formatted)
+      ? <pre style={{ whiteSpace: 'pre-wrap' }}>{formatted}</pre>
```

Comment:
If `formatted` is `undefined` (format failed), we should display the plain body.